File: battleships/src/Model/ResultOfAttack.vb
''' <summary>
''' The result of an attack.
''' </summary>
Public Enum ResultOfAttack
    ''' <summary>
    ''' The player hit something
    ''' </summary>
    Hit

    ''' <summary>
    ''' The player missed
    ''' </summary>
    Miss

    ''' <summary>
    ''' The player destroyed a ship
    ''' </summary>
    Destroyed

    ''' <summary>
    ''' That location was already shot.
    ''' </summary>
    ShotAlready

    ''' <summary>
    ''' The player killed all of the opponents ships
    ''' </summary>
    GameOver
End Enum

File: battleships/src/Model/ResultOfAttack.cs
// '' <summary>
// '' The result of an attack.
// '' </summary>
public enum ResultOfAttack {
    Hit, // '' The player hit something

    Miss, // '' The player missed

    Destroyed, // '' The player destroyed a ship

    ShotAlready, // '' That location was already shot.

    GameOver, // '' The player killed all of the opponents ships
}